### ISSUE TITLE
Use `0.16.0-rc.2` branch of box-scripts

### ISSUE THREAD

**Issue by mikeschinkel** [wplib/wplib-box#344]
@MickMakes The current box scripts has a complex directory structure that no longer makes sense. I created a [`flatten` branch](https://github.com/wplib/box-scripts/tree/flatten) to resolve that.

Please incorporate in the upcoming RC?

**Comment by MickMakes** [comment]
OK. I'll look at merging with 0.16.

**Comment by mikeschinkel** [comment]
@MickMakes I have another branch that I think I will replace this with and I am almost finished working on it.  It is the `database` branch.  I'd also like you to test it and do a bit of bash code review?

**Comment by mikeschinkel** [comment]
@MickMakes I have changed the name of this ticket to reference `database` which includes the changes in `flatten`.

This adds a `box database` command following your lead for `box container`.  The primary commands are:

```
box database chunk  <sql_file>
box database unchunk <sql_file>
box database backup [<sql_file>]
box database import  [<sql_file>]
```
I have tested all but that last one reasonably well, but cannot test the last one because it does not use Docker yet.  We need that fixed too.
